src/components/DeliverableView/useFetchDeliverable.ts

```diff
@@ -33,16 +30,20 @@ export default function useFetchDeliverable({ deliverableId, projectId }: Props)
   const snackbar = useSnackbar();
   const history = useHistory();
   const dispatch = useAppDispatch();
-  const deliverableResult = useAppSelector(selectDeliverableFetchRequest(deliverableId));
-  const deliverableDataResult = useAppSelector(selectDeliverable(deliverableId));
```

Comment:
Thanks for the change. Curious why this earlier version didn't work. The dispatch should have triggered a new fetch and data update in the store. I'm probably missing something.

Reply:
It also might be confusing because the original change to the service was done in a previous PR

Reply:
@karthikbtf here's my crack at that https://github.com/terraware/terraware-web/pull/2289